db/migrate/20221026193322_add_user_ref_to_comment.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] The same here 🤔 The Comments belongs-to user through the `'author_id'` column of the Comment table. Now, we need to show that, the  `'author_id'` reference the User class modal. Kindly write the query like this so the foreign key name will be `author_id`. 👇🏻 :

```ruby
   add_reference :comments, :author, null: false, foreign_key: { to_table: :users }
```